Libraries/Inspector/ElementProperties.js

```diff
@@ -26,27 +25,16 @@ const openFileInEditor = require('openFileInEditor');
 
 import type {DangerouslyImpreciseStyleProp} from 'StyleSheet';
 
-class ElementProperties extends React.Component<{
+type Props = $ReadOnly<{|
   hierarchy: Array<$FlowFixMe>,
-  style?: DangerouslyImpreciseStyleProp,
-  source?: {
-    fileName?: string,
-    lineNumber?: number,
+  style?: ?DangerouslyImpreciseStyleProp,
```

Comment:
You should change this up, so instead of importing and using `DangerouslyImpreciseStyleProp`, it uses `ViewStyleProp`